src/patch_parser.py

```diff
@@ -415,34 +433,41 @@ def _does_match_end_of_super_chunk(lines: InputSource) -> bool:
 def _parse_patch_file_unchanged_chunk(
         lines: InputSource,
         gerrit_orig_line: int,
-        gerrit_new_line: int) -> Tuple[int, int, PatchFileChunkLineMap]:
+        gerrit_new_line: int,
```

Comment:
I was originally thinking we could move these gerrit_orig/new_line fields into HunkParserState.
Wdyt?

Reply:
It's fine, I just left them there because of the original design, but I can move them over.